main.py

```
@@ -384,12 +485,31 @@ async def step_pwd(message: Message, state: FSMContext):
+        if not message.text:
+            await message.answer("❌ Пожалуйста, отправьте текстовое сообщение.")
+            return
         email = message.text.strip()
         
-        if not validate_email(email):
-            await message.answer(T["err_mail"])
+        try:
+            await message.delete()
+        except Exception:
+            pass
+        # Улучшенная валидация email с проверкой провайдера
```

Ruff (0.14.10)

Comment contains ambiguous `с` (CYRILLIC SMALL LETTER ES). Did you mean `c` (LATIN SMALL LETTER C)?

(RUF003)